engine/src/main/java/org/terasology/rendering/dag/WireframeStateChange.java

```diff
@@ -16,11 +16,20 @@
 package org.terasology.rendering.dag;
 
 
+import static org.lwjgl.opengl.GL11.GL_FILL;
+
 /**
  *
  */
 public class WireframeStateChange extends AbstractStateChange<Integer> {
+    private static WireframeStateChange defaultInstance = new WireframeStateChange(GL_FILL);
+
     WireframeStateChange(Integer value) {
```

Comment:
Thinking about usage in a node here: the way it works right now is that a node would have to list in its desired state an instance of SetWireframe created with 

`desiredStateChanges.add(SetWireframe(GL_LINE));`

The problem is that GL_LINE really is a polygon mode and a user of this class has to go through a couple of mental hoops to understand the relationship between wireframe and the polygon mode.

What would be much more readable is:

`desiredStateChanges.add(SetWireframe(enabled));`

SetWireframeStateChange(enabled)  wouldn't read as nicely by the way.